lib/inc/drogon/drogon_test.h

```diff
@@ -174,6 +146,7 @@ inline std::string attemptPrint(T&& v)
 template <>
 inline std::string attemptPrint(const std::nullptr_t& v)
 {
+    (void)v;
```

Comment:
AFAIK, you do not need to take an argument name here
```suggestion
inline std::string attemptPrint(const std::nullptr_t&)
{
```